client/src/components/App.jsx

```diff
@@ -12,6 +12,7 @@ import Modal, { ModalProvider, BaseModalBackground } from "styled-react-modal";
 const FadingBackground = styled(BaseModalBackground)`
   opacity: ${(props) => props.opacity};
   transition: all 0.3s ease-in-out;
+  z-index: 2000;
```

Comment:
wow you REALLY want this to stand out in front lol

Reply:
Lools yea.... its cuz the hardcode react-multi-carosul arrow has an zindex of 1000 :/   Lols will def look into swipper next project